pkg/cmd/server/start/start_master.go

```diff
@@ -535,6 +543,17 @@ func startControllers(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) erro
 		controllerManagerOptions = kc.ControllerManager
 	}
 
+	// TODO: Make the resync period configurable via the master config.
+	resyncPeriod := 2 * time.Minute
```

Comment:
To be honest, with different caches in the controller, I don't really care
about it. It's just the default we have now.

On Thu, Jun 16, 2016 at 4:32 PM, David Eads <notifications@github.com>
wrote:

> In pkg/cmd/server/start/start_master.go
> <https://github.com/openshift/origin/pull/9002#discussion_r67355391>:
>
> > @@ -535,6 +543,17 @@ func startControllers(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) erro
> >  		controllerManagerOptions = kc.ControllerManager
> >  	}
> >
> > +	// TODO: Make the resync period configurable via the master config.
> > +	resyncPeriod := 2 * time.Minute
>
> Yeah, this is a general problem for us. Do you need it this tight by
> default?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9002/files/489ed7628ca7ef89d8fe424ffb8725b686eebd76#r67355391>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADuFf6KhEzy8n3g2R8G2TW63KlizKPK3ks5qMV6IgaJpZM4IliRU>
> .
>